pom.xml

```diff
@@ -639,6 +642,93 @@
                 <allowConventionalDistJar>true</allowConventionalDistJar>
             </properties>
         </profile>
+        <profile>
+            <id>scala-2.12</id>
+            <properties>
+                <scala.version>2.12.15</scala.version>
+            </properties>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                    </plugins>
+                </pluginManagement>
+            </build>
+        </profile>
+        <profile>
+            <id>scala-2.13</id>
+            <properties>
+                <scala.version>2.13.8</scala.version>
+                <scala.binary.version>2.13</scala.binary.version>
+            </properties>
+            <dependencyManagement>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.scalatest</groupId>
+                        <artifactId>scalatest_${scala.binary.version}</artifactId>
+                        <version>3.2.16</version>
```

Comment:
In cases like this can we introduce a property that changes values between profiles instead of copying the whole dependency definition?